runtime/src/bank.rs

```diff
@@ -2985,6 +2985,7 @@ impl Bank {
         self.slots_per_year = genesis_config.slots_per_year();
 
         self.epoch_schedule = genesis_config.epoch_schedule.clone();
+        self.transaction_processor.epoch_schedule = genesis_config.epoch_schedule.clone();
```

Comment:
Is there any test for this line? I thought the EpochSchedule was correctly configured when we create the TransactionBatchProcessor using the `new` function.

Reply:
It'll only be a problem when the epoch schedule is first read from the genesis block. Afterwards it gets stored as part of the bank in the snapshot. So we may not hit this issue in real life. 
I don't think there's a test. Good call on that. I'll add one as part of the PR.

Reply:
Added the test. It fails with the following without this PR:
```
assertion `left == right` failed
  left: EpochSchedule { slots_per_epoch: 432000, leader_schedule_slot_offset: 432000, warmup: true, first_normal_epoch: 14, first_normal_slot: 524256 }
 right: EpochSchedule { slots_per_epoch: 8192, leader_schedule_slot_offset: 100, warmup: true, first_normal_epoch: 8, first_normal_slot: 8160 }
```